### DISCUSSION TITLE
8gb/8gb connection breaks the cli

### DISCUSSION THREAD

**Discussion by Skyeci73** [alexjustesen/speedtest-tracker#1901] [Q&A]
Having used this excellent app for a fair while my isp gave me the opportunity to have a go with their 8gb/8gb service which effectively gives you 7gb/7gb. The cli struggles to record the upload correctly.
My download sits at 8100 but the upload is recorded at 4-5gb. I've run some iperf tests which are above this figure.
I don't suppose there is any resolution for this..
Example
https://www.speedtest.net/result/c/65fc6efb-0071-4da3-bcec-63886d553903

**Comment by svenvg93** [comment]
I'm afraid this one of the "downsides" of having a 8Gbit connection, reaching it in tests is a challenge. I see the same in a 8GB test connection when using Ookla, the download is always round the 6/7Gbit, where the upload is more around the 5/6Gbit when using the ISP Ookla server.

For example the test below:
ISP Ookla server:
https://www.speedtest.net/result/c/149fb104-d01c-43c2-abcc-bad3e158b9a0

Third party Ookla Server: 
https://www.speedtest.net/result/c/3c34105c-d6a3-4267-8da9-49b94f18472f

I'm not sure if this a limition on the Ookla CLI with the amount of connections it uses or more on the network/server part. From the application point of view there is nothing we can do about this. As we just leverage the Ookla CLI.

**Reply by Skyeci73** [reply]
Thanks  . I think you are probably correct.  The only way I can max it out on the upload for testing purposes is by iperf3 against a net based server...

Just need you guys to build iperf3 in now 😂😂😂
